### ISSUE TITLE
dynamic region

### ISSUE THREAD

**Issue by whiskywaffel** [achannarasappa/ticker#6]
There isn't an option for changing the region. Would be nice to use an argument in the ticker.yaml

![image](https://user-images.githubusercontent.com/37911718/106451217-86821780-6486-11eb-8560-39969567c2ba.png)

**Comment by achannarasappa** [comment]
Hi @whiskywaffel

Would you mind sharing how this would help you / the use case?

**Comment by pgrunm** [comment]
For example for users from other continents like me. I'm from the EU and right now we can't change the currency from USD to EUR. I don't know if this comes from the region, but it would be awesome if there would be a setting to change the displayed currency.

**Comment by achannarasappa** [comment]
@pgrunm that really helpful context thank you

I've tested the API with this call:
` http "https://query1.finance.yahoo.com/v7/finance/quote?lang=en-GB&region=GB&corsDomain=finance.yahoo.com&symbols=NET" | jq '.quoteResponse.result[0].currency'`
From this, I found that the currency returned is still `USD` which indicates that the currency is a property of the exchange rather than the region from the request. If alternate currencies are to be supported, conversion might need to be done on the client side which brings up some data accuracy concerns in my mind as exchange rates fluctuate throughout the day and for certain stocks, an even slightly inaccurate exchange rate could yield significantly inaccurate quote prices.

In any case, further exploration of the API for ways to convert currency and client side currency conversion is warranted since I've heard this request a few times now.
